docs/layers.md

```diff
@@ -743,6 +743,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 * `max_zoom`: a suggested maximum zoom beyond which the place should not be visible. Currently neighbourhoods only, from Who's On First.
 * `is_landuse_aoi`: Currently neighbourhoods only, from Who's On First
 * `kind_detail`: the original value of the OSM `place` tag and Natural Earth `featurecla`, see below.
+* `wikidata`: when present, the [Wikidata](https://www.wikidata.org) ID corresponding to this feature.
```

Comment:
Let's export these as `wikidata_id` so it's clear what that it's a concordance value not a web link.